### PR TITLE
added quick start section

### PR DESCRIPTION
**Pull request by jamesnaftel**

### CODE REVIEW

**Review by zombiezen — APPROVED** [review]
Thanks James!

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <img alt="YourBase" src="images/Logo-Horiz-On-White@1x.jpg" width="384" height="112">
 
+# Quick Start Acceleration
```
Comment (zombiezen) [inline docs/README.md:3]:
```suggestion
# Test Acceleration
```

**Review by zombiezen — CHANGES_REQUESTED** [review]
Actually, wait. I don't think we're using "Quick Start" as the external naming.

**Comment by jamesnaftel** [comment]
You are correct!  Brain fart...

**Review by zombiezen — APPROVED** [review]

**Pull request merged**